### ISSUE TITLE
[RFC] [EntityContext] JSON for arrays

### ISSUE THREAD

**Issue by Nek-** [KnpLabs/FriendlyContexts#226]
Hi,

Currently to specify an array we need to write it this way:
```
Item1, Item2, Item3
```

This is a problem mainly because it doesn't support multi-dimensions. I suggest to also support JSON so we'll be able to write array this way in build in fixtures:
```
{"cat1": ["item1", "item2"], "cat2": ["item3"]}
```

**Comment by pimolo** [comment]
I don't think that JSON would be the answer, since JSON can be supported (for instance by MySQL) as a format as it is. How would you distinct real JSON and PHP arrays? :confused:
What about:
```
["cat1": ["item1", "item2"], "cat2": ["item3"]]
```
?

**Comment by Nek-** [comment]
@pimolo yes and it's pretty easy to go on. Do it if you want :).
